NOnion/Directory/TorDirectory.fs

```diff
@@ -237,15 +237,14 @@ type TorDirectory =
                 self.NetworkStatus <- NetworkStatusDocument.Parse response
         }
 
-    static member Bootstrap
-        (nodeEndPoint: IPEndPoint)
+    static member BootstrapWithGuard
         (cacheDirectory: DirectoryInfo)
+        (guard: TorGuard)
```

Comment:
Please preserve the order of the arguments.

Reply:
done